### DISCUSSION TITLE
How to show default log from GetX, when using flutter run instead using debugging ?

### DISCUSSION THREAD

**Discussion by lypsisrudiansyah** [jonataslaw/getx#2618] [Q&A]
Log showing on debugging
![image](https://user-images.githubusercontent.com/52348378/202082183-f88a39d2-6209-4425-8d0d-6a476949389b.png)

Log not showing on flutter run
![image](https://user-images.githubusercontent.com/52348378/202082187-caff7765-eba0-427f-9aa2-8dc07678e73a.png)

**Comment by anasfik** [comment]
Hi,
since Getx's default logger is implemented like this:
```/// default logger from GetX
void defaultLogWriterCallback(String value, {bool isError = false}) {
  if (isError || Get.isLogEnable) developer.log(value, name: 'GETX');
} 
```
it's based on the developer's log() method, which only appears in the debug mode, so you can't show it actually in the release mode unless you use logging with a normal print() method.
